scripts/src/main/resources/scripts/command/docker

```diff
@@ -72,6 +126,65 @@ function doIsDockerInstalled() {
   fi
 }
 
+#OS independent check if Rancher Desktop is installed.
+function doIsRancherInstalled() {
+    if doIsWindows
+    then
+      touch tmp-res-install-query rd-install-test.ps1
+      # shellcheck disable=SC2016,2028
+      echo 'if($env:Path -like "*Rancher Desktop*") { set-content .\tmp-res-install-query "installed"} else {set-content .\tmp-res-install-query "not"}' > rd-install-test.ps1
+      powershell.exe -Command ".\rd-install-test.ps1"
+      installvar=$(awk 'NR=1' tmp-res-install-query)
+      rm tmp-res-install-query rd-install-test.ps1
+      if [ "${installvar}" = "installed" ]
+      then
+        return 
+      else
+        return 255
+      fi
+    else
+      if command -v rancher-desktop &> /dev/null
+      then
+        return
+      else
+        return 255
+      fi
+    fi
+}
+
+function doInstallVpnkit() {
+  #get latest version of the vpnkit and download its tar.gz file
+  partialUrl=$(curl -Ls -o /dev/null -w %{url_effective} https://github.com/sakai135/wsl-vpnkit/releases/latest)
+  VPNversion=${partialUrl##*/}
+  downloadUrl="https://github.com/sakai135/wsl-vpnkit/releases/download/${VPNversion}/wsl-vpnkit.tar.gz"
+  curl -fL "${downloadUrl}" -o "wsl-vpnkit.tar.gz"
+  touch activate-vpnkit
+  touch vpnkit-setup.ps1
+
+  #at first write a bash-script
+  touch activate-vpnkit
```

Comment:
this command is duplicated with line 161 and you would not even need this if you use `>` instead of `>>` in line 166.
But if we want some nice-to-have perfection tidy up here, we can also do it in the next PR after merge.